### DISCUSSION TITLE
配置 gpu 可以通过 id 配置吗，目前看代码不支持通过 id，只能用 num_gpus

### DISCUSSION THREAD

**Discussion by amulil** [open-compass/opencompass#282] [View all discussions]
rt.

**Comment by Leymore** [comment]
You can try command like `CUDA_VISIBLE_DEVICES=0,1,3 python run.py blabla`

Implement code is here:

https://github.com/InternLM/opencompass/blob/b2d602f42bac783bc30ef8e903177d7ab378a0de/opencompass/runners/local.py#L77-L81
